pinot-controller/src/main/resources/app/utils/PinotMethodUtils.ts

```diff
@@ -746,6 +761,113 @@ const deleteInstance = (instanceName) => {
   });
 };
 
+const getAllPeriodicTaskNames = () => {
+  return getPeriodicTaskNames().then((response)=>{
+    return { columns: ['Task Name'], records: response.data.map(d => [d]) };
+  });
+};
+
+const getAllTaskTypes = async () => {
+  const finalResponse = {
+    columns: ['Task Type', 'Num Tasks in Queue', 'Queue Status'],
+    records: []
+  }
+  await new Promise((resolve, reject) => {
+    getTaskTypes().then(async (response)=>{
+      if (_.isArray(response.data)) {
```

Comment:
Let's fix lodash imports in this file. Importing the entire library is not a good idea. Let's change it to `import { a, b, c } from 'lodash';`